annif/templates/home.html

```diff
@@ -112,20 +133,51 @@ <h2 v-if="problem.hasTitle()"><% problem.title %></h2>
   },
   template: '<div>\
   <div class="form-group">\
-    <label for="project">Project (vocabulary and language):</label>\
+    <label for="project">Project (vocabulary and language)</label>\
+    <div class="select-wrapper">\
     <select class="form-control" id="project" @input="handleProjectSelected">\
       <option v-for="project in projects" v-bind:value="project.project_id"><% project.name %></option>\
+    </div>\
```

Comment:
I think the `div` is closing the `select`. Probably need to move one line down?

Reply:
Good catch! The order was a bit off